app/assets/javascripts/discourse/templates/components/d-navigation.hbs

```diff
@@ -12,6 +12,13 @@
   {{category-notifications-button value=category.notification_level category=category}}
 {{/if}}
 
+{{plugin-outlet name="alongside-create-topic-button"
```

Comment:
I would suggest calling it `before-create-topic-button` but otherwise it's all good.

Reply:
Agreed, thanks :)